h/static/scripts/directives/markdown.coffee

```diff
@@ -336,7 +336,7 @@ markdown = ['$filter', '$sanitize', '$sce', '$timeout', ($filter, $sanitize, $sc
     # React to the changes to the input
     inputEl.bind 'blur change keyup', ->
       ctrl.$setViewValue inputEl.val()
-      scope.$digest()
+#      scope.$digest()  # TODO: this shouldn't be a problem
```

Comment:
No, now I can see it in chrome but not always.. it seems the error only comes when you select a specific part. Interesting, for example, in our welcome page I select "started" to get it, but selecting "Getting" in chrome does not trigger it. (So the two browsers are behaving a bit different, but we can have the error in chrome too)